translate/v23tov30/v23tov30.go

```diff
@@ -347,12 +348,50 @@ func translateFilesystems(fss []old.Filesystem, m map[string]string) (ret []type
 		if f.Mount == nil {
 			f.Mount = &old.Mount{}
 		}
+
+		wipe := util.BoolP(f.Mount.WipeFilesystem)
+		label := f.Mount.Label
+		uuid := f.Mount.UUID
+
+		// If `wipe` is set to `false` but we have a `"create": {...}` section, we try
+		// to convert it.
+		if wipe == nil && f.Mount.Create != nil {
+			wipe = util.BoolP(f.Mount.Create.Force)
```

Comment:
If `Force` is true, it's safe to set `WipeFilesystem` true.  But if `Mount.Create` is non-nil and `Force` is false, I think we should fail the translation.  In that situation, the config is asking for the FS to be created if it doesn't exist, and for Ignition to fail otherwise.  For the case where the filesystem exists and matches the specified parameters (format etc.), Ignition 2.x can't deliver those semantics: provisioning would succeed instead.

Reply:
Let's discuss that change in a separate issue.  I don't think it's relevant here, though?  The problem here isn't the mountpoint, but what to do if the specified `Device` already has a filesystem superblock on it.

Reply:
Ok, the issue is created in here: https://github.com/coreos/ign-converter/issues/30

> The problem here isn't the mountpoint, but what to do if the specified Device already has a filesystem superblock on it.
Got it.

Reply:
Okay, cool.  So let's add an error return in `Check2_*` if `f.Mount.Create != nil && !f.Mount.Create.Force`, and a comment here referring to that.